src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -167,10 +178,60 @@ export default ({
   useEffect(() => {
     const viewer = cesium.current?.cesiumElement;
     if (!viewer || viewer.isDestroyed()) return;
-
     viewer.scene.requestRender();
   });
 
+  const handleDragLayer = useCallback(
+    (e: Entity, position: Cartesian3 | undefined, _context: Context): boolean | void => {
+      onDragLayer?.(e.id, convertCartesian3ToPosition(position));
+    },
+    [onDragLayer],
+  );
+
+  const handleDraggingLayer = useCallback(
+    (e: Entity, position: Cartesian3 | undefined, _context: Context): false | void => {
+      onDraggingLayer?.(e.id, convertCartesian3ToPosition(position));
+    },
+    [onDraggingLayer],
+  );
+
+  const handleDropLayer = useCallback(
+    (e: Entity, position: Cartesian3 | undefined, _context: Context): boolean | void => {
+      onDropLayer?.(e.id, convertCartesian3ToPosition(position));
+      if (cesium.current?.cesiumElement?.scene.screenSpaceCameraController) {
+        cesium.current.cesiumElement.scene.screenSpaceCameraController.enableRotate = true;
+      }
+    },
+    [onDropLayer],
+  );
+
+  const convertCartesian3ToPosition = (
+    pos?: Cartesian3,
+  ): { lat: number; lng: number; height: number } | undefined => {
+    if (!pos) return;
+    const cartographic =
+      cesium.current?.cesiumElement?.scene.globe.ellipsoid.cartesianToCartographic(pos);
+    if (!cartographic) return;
+    return {
+      lat: CesiumMath.toDegrees(cartographic.latitude),
+      lng: CesiumMath.toDegrees(cartographic.longitude),
+      height: cartographic.height,
+    };
+  };
+
+  //Enable Drag and Drop Layers
+  const DELAY = 100;
```

Comment:
Sorry super small, but also wanted to hear your opinion on this.
Is this really necessary? I know if where it is being used is not clear doing this will make it clear what the value is. But below where this is used the key is `dragDelay`, which is clear even if it is `dragDelay: 100`.
Don't need to change but would like to hear if you think this is now overkill.

Reply:
I don't have a strong reason why I declared this as a variable. 
This is a kind of config value and I thought in the future we might move this variable to another config file if needed. To do that, I chose this.